### PR TITLE
Add velocity task.

### PR DESCRIPTION
**Pull request by zacyu**
Mering this to my working branch (pending review for the hold position task). The velocity task accelerates the drone to a given velocity in x-y axes and set the z-velocity.

### CODE REVIEW

File: roombasim/pittras/task/velocity_task.py
Comment (amiller27) [inline roombasim/pittras/task/velocity_task.py]: These should be different constants than the ones used for the xyz translation task

File: roombasim/pittras/task/velocity_task.py
Comment (amiller27) [inline roombasim/pittras/task/velocity_task.py:46]: You need to check that the requested acceleration is actually within the drone's acceleration limits.  This should probably also be done inside the `control` function, but that's not for this PR.

File: roombasim/pittras/task/velocity_task.py
Comment (amiller27) [inline roombasim/pittras/task/velocity_task.py]: Do we need this?

File: roombasim/pittras/config.py
Comment (amiller27) [inline roombasim/pittras/config.py]: This seems high, maybe more like `0.05` or `0.1`?

**Review by amiller27 — COMMENTED** [review]
Looks good except for some minor things.  We typically don't do this kind of merge structure, but it's up to you guys how you want to do it.  Typically this PR would rebase onto `master` once `zyu_tasks` is merged, and would then merge directly into `master`.  Like I said, you guys can decide how you want to do things.

**Comment by zacyu** [comment]
@amiller27 Yeah, I am planning to rebase it onto `mater` later. Doing it this way so I am not blocked by #7

File: roombasim/pittras/config.py
Comment (zacyu) [reply roombasim/pittras/config.py]: Sure :)

File: roombasim/pittras/task/velocity_task.py
Comment (zacyu) [reply roombasim/pittras/task/velocity_task.py]: Done

File: roombasim/pittras/task/velocity_task.py
Comment (zacyu) [reply roombasim/pittras/task/velocity_task.py]: Nice catch. I didn't realize that the z axis is simply velocity at first.

File: roombasim/pittras/task/velocity_task.py
Comment (zacyu) [reply roombasim/pittras/task/velocity_task.py:46]: Ack'd

**Review by zacyu — COMMENTED** [review]
PTAL

File: roombasim/pittras/task/velocity_task.py
Comment (hgarrereyn) [inline roombasim/pittras/task/velocity_task.py:12]: Need to add `VelocityTask` to roombasim.config.TASKS

File: roombasim/pittras/task/velocity_task.py
Comment (hgarrereyn) [inline roombasim/pittras/task/velocity_task.py:40]: `hold_xy` should be `target_xy` I think

File: roombasim/pittras/task/velocity_task.py
Comment (hgarrereyn) [inline roombasim/pittras/task/velocity_task.py:46]: why are you using `control_xy * delta` rather than `control_xy`?

**Review by hgarrereyn — CHANGES_REQUESTED** [review]

File: roombasim/pittras/task/velocity_task.py
Comment (mbilker) [reply roombasim/pittras/task/velocity_task.py:46]: You already multiply by `delta` on line 42, which goes into `control_xy` on line 43. Is it supposed to be multiplied again on line 46?

File: roombasim/pittras/ai/velocity_demo_controller.py
Comment (mbilker) [inline roombasim/pittras/ai/velocity_demo_controller.py:21]: Probably doesn't matter for the sake of the demo controller, I am questioning whether the callback method should be a method in `VelocityDemoController` rather than outside the class.

**Review by mbilker — COMMENTED** [review]
I have some comments, but @hgarrereyn 's comments are the same as my concerns.

**Comment by asaba96** [comment]
Also what's going on with this one?

**Comment by hgarrereyn** [comment]
These changes are present in #12

**Pull request closed without merging**
